src/pages/Suggestions/styles.css

```diff
@@ -13,3 +21,45 @@
   display: flex;
   flex-direction: column;
 }
+
+.modal-container {
+  background-color: var(--color-background);
+  border-color: var(--color-white);
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 1px;
+  color: var(--color-white);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 25px;
+  height: 50%;
+  width: 50%;
+}
+
+.field-container {
+  display: flex;
+  flex-direction: column;
+  padding: 10px 20px;
+}
+
+.field-label {
+  font-weight: 800;
+  padding-bottom: 10px;
+}
+
+.add-suggestion-button {
+  margin: 10px auto;
+  max-width: 12rem;
```

Comment:
do not mix rem with px

Reply:
thanks I think I dont need this, I can delete it 😄